rfd/0131-adminitrative-actions-mfa.md

```diff
@@ -82,10 +96,52 @@ Instead, in the first major version of this feature (likely Teleport 14), it wil
 always default to `off`. In the next major version (Teleport 15), the defaults
 above will be used.
 
+#### Built-in Roles
+
+Built-in roles will not require MFA to complete admin actions. This includes
+the `Admin` role which is used by `tctl` when performing operations directly
+on the Auth Server.
+
+#### Automated Use Cases
+
+In order to support automated use cases such as the
+[Teleport Terraform Provider](https://goteleport.com/docs/management/guides/terraform-provider/)
+or [Access Request Plugins](https://goteleport.com/docs/access-controls/access-request-plugins/),
+we need to provide a way for specific users to bypass MFA for admin actions.
+
+We will introduce a new RBAC verb, `bypass-mfa`, which can be included in a
```

Comment:
WDYT about adding a toggle to the role that says it's a robot, instead of a new verb? I know I'm guilty of new verbs, but since then I've learned folks are not too keen on those, and I get the rationale.

`bypass-mfa` isn't much a verb, as it isn't something you do, but instead something you are allowed to skip? A robot toggle could come in handy as a proxy for other behaviors too.

Reply:
Noted, see the thread above. I am concerned about wrapping this bypass into a larger encompassing `robot` toggle though. I'll have to think about it more.

Reply:
Hmm I agree verb is a bit confusing. How about a role option? We already have `require_session_mfa`, could do something similar for this as well?

Reply:
I decided to add the `robot` role option as recommended.